app/client/components/contactUs/contactUsForm.tsx

```diff
@@ -142,7 +142,11 @@ export const ContactUsForm = (props: ContactUsFormProps) => {
         ...formValidationState.subjectLine,
         isValid: isSubjectLineValid
       },
-      details: { ...formValidationState.fullName, isValid: isDetailsValid }
+      details: { ...formValidationState.fullName, isValid: isDetailsValid },
```

Comment:
Should it be this instead?
```suggestion
      details: { ...formValidationState.details, isValid: isDetailsValid },
```